source/nl/1.0.0/index.html.haml

```diff
@@ -267,27 +267,27 @@ version: 1.0.0
     %a.anchor{ href: "#rewrite", aria_hidden: "true" }
     Mag je een changelog aanpassen/herschrijven?
 
-  %p
-    Natuurlijk. Er zijn goede redenen om een changelog te verbeteren.
-    Ik open regelmatig een pull request om missende releases toe te
-    voegen aan open source projecten met een slecht onderhouden changelog.
+    %p
+      Natuurlijk. Er zijn goede redenen om een changelog te verbeteren.
+      Ik open regelmatig een pull request om missende releases toe te
+      voegen aan open source projecten met een slecht onderhouden changelog.
 
-  %p
-    Het kan ook zo zijn dat je ontdekt dat je een belanrijke aanpassing niet
-    vermeld hebt in je changelog. Het is dan natuurlijk zaak om dit alsnog
-    in je changelog te vermelden.
+    %p
+      Het kan ook zo zijn dat je ontdekt dat je een belanrijke aanpassing niet
```

Comment:
Typo: belanrijke => belangrijke

It appears more often in the document (e.g. line 290 and 299) I'd suggest to just _search and replace_ "belanrijk" with "belangrijk"

Reply:
Good one, done!